.github/workflows/feature_5_legacy_docs.yml

```diff
@@ -0,0 +1,65 @@
+name: Develop > Legacy Tiny 5 Docs
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**'
+    branches:
+      - 'feature/5/DOC-1640'
```

Comment:
This was very much intentional, as we can't use `feature/*` or `hotfix/*` as then it'd run on any of our Antora branches as well. As such, the main aim here was to test this is working.

If we want to allow branch builds for the old jekyll docs then we'll need to come up with a specific branch structure that won't conflict.

Reply:
I can think of other ways, but they aren't worth the hassle. We only need the bare minimum until the 5x Antora branch is ready.

Reply:
Yeah I also had some options (e.g. using the old docker image wercker used) but I also just didn't see the point atm.

Reply:
Oops forgot what this comment was about. Ignore the above largely sorry 😞 

Anyways, yeah the only time we might need it is for the 5.10.x patches until the new site is ready, so I'm fine with this manual process for now.